common/trtexec_vendor/CMakeLists.txt

```
@@ -28,7 +28,8 @@ if(TENSORRT_VERSION VERSION_LESS 8.2.1)
 endif()
 
 set(TRTEXEC_DEFAULT_BIN /usr/src/tensorrt/bin/trtexec)
-if(NOT EXISTS TRTEXEC_DEFAULT_BIN)
+if(NOT EXISTS ${TRTEXEC_DEFAULT_BIN})
+  message(STATUS "${TRTEXEC_DEFAULT_BIN} not found, download and build from source...")
   include(FetchContent)
   fetchcontent_declare(tensorrt
     GIT_REPOSITORY https://github.com/NVIDIA/TensorRT
```

We get this error:
- https://github.com/autowarefoundation/autoware.universe/actions/runs/6884112074/job/18941486981?pr=5464#step:6:309

Looking at the docs: https://cmake.org/cmake/help/latest/module/FetchContent.html

```suggestion
    GIT_REPOSITORY https://github.com/NVIDIA/TensorRT.git
```

We might need to modify this this way.

---

We still get the same error after adding ".git". I guess there is another issue to solve.

---

I re-run the cmake locally, it builds ok (cmake fetch and then build).